app/test_app/functional/api/test_metrics_api.py

```diff
@@ -233,11 +233,11 @@ def get_metrics(metric_type):
         total_spent_val = 25
 
     elif metric_type == 'credit_transfer' or metric_type == 'all':
-        assert returned_stats['daily_disbursement_volume']['aggregate']['ORGANISATION'] == 300
+        assert returned_stats['daily_disbursement_volume']['aggregate']['ORGANISATION'] == 700
         assert returned_stats['daily_transaction_volume']['aggregate']['USER'] == 150
         assert returned_stats['exhausted_balance'] == 0
         assert returned_stats['has_transferred_count'] == 2
-        assert returned_stats['total_distributed'] == 300
+        assert returned_stats['total_distributed'] == 700
```

Comment:
**Disbursement total is now 700**, as original 300 was the custom disbursement in lines 29 and 37 of this file and 400 is the default disbursement amount of `create_organisation`

Reply:
This is outdated but still applies in the metrics JSON objects